openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
```

Comment:
Cale is right: the goal was to have a compact nick name that we could use for textual reports. We don't currently use them much, but we should stick to the original goal, since the other repos have short nicks also.